apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -365,6 +365,8 @@ export default function GlobalStyle () {
               }
 
               .ant-table-tbody > tr.condo-table-expandable-row {
+                font-weight: ${DEFAULT_STRONG_TEXT_FONT_WEIGHT};
```

Comment:
How does this relate relate to this PR?

Reply:
just made font-weight in expandable table row (permissions table) as in figma layouts